contracts/src/v0.8/functions/dev/1_0_0/RouterBase.sol

```diff
@@ -181,20 +176,17 @@ abstract contract RouterBase is IRouterBase, Pausable, ITypeAndVersion, Confirme
     }
   }
 
-  /**
-   * @inheritdoc IRouterBase
-   */
+  // @inheritdoc IRouterBase
   function updateContracts() external override onlyOwner {
     if (block.number < s_proposedContractSet.timelockEndBlock) {
       revert TimelockInEffect();
     }
     // Iterations will not exceed MAX_PROPOSAL_SET_LENGTH
     for (uint256 i = 0; i < s_proposedContractSet.ids.length; ++i) {
       bytes32 id = s_proposedContractSet.ids[i];
-      address from = s_route[id];
       address to = s_proposedContractSet.to[i];
+      emit ContractUpdated({id: id, from: s_route[id], to: to});
```

Comment:
```suggestion
      emit ContractUpdated({id: s_proposedContractSet.ids[i], from: s_route[id], to: s_proposedContractSet.to[i]});
```

Reply:
This is done for size reasons. `s_proposedContractSet.ids[i]` and `s_proposedContractSet.to[i]` are set to variables because they are referenced in two places